### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by drwitter**
Datums ingeladen, kleine bug fixes toegepast

### CODE REVIEW

File: app/assets/javascripts/search_dashboard.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      $scope.products = data;
+    });
+  });
+  
+  $scope.submitform = function () {
+    window.location.href = $scope.searchTag;
+  }
+  $scope.getCategory = function (msg) {
+    for(var i = 0; i <= categories.length ; i ++){
+      if(i == categories.length){
```
Comment (robinsiep) [inline app/assets/javascripts/search_dashboard.js]: is it even possible for this check to fail? It seems to only iterate if there are categories

File: app/assets/javascripts/search_dashboard.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        if(msg == 'Processors'){
+          return 'cpu';
+        }else if(msg == 'CPU-koelers'){
```
Comment (robinsiep) [inline app/assets/javascripts/search_dashboard.js]: What are these checks for?

Comment (drwitter) [inline app/assets/javascripts/search_dashboard.js]: Aangezien niet alle categorieen benoemd worden met locale{nl_NL} moeten sommige handmatig zo gebeuren. Anders gaat hij fout

Comment (sanderdewinter) [inline app/assets/javascripts/search_dashboard.js]: Build failed, er is iets fout dan in je code :(

File: app/assets/javascripts/search_dashboard.js (hunk 1 above)
Comment (robinsiep) [inline app/assets/javascripts/search_dashboard.js]: If it's not possible, remove the if statement.

File: app/assets/javascripts/search_dashboard.js (hunk 2 above)
Comment (robinsiep) [inline app/assets/javascripts/search_dashboard.js]: Add a locale in the database, don't check for it.  Also all categories do contain a locale now

File: app/controllers/dashboard_controller.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class DashboardController < ApplicationController
+
+  def getData
+
+    @json_data = JSON.parse(Net::HTTP.get(URI.parse('http://95.85.12.99:6543/category/'+params[:cat_name]+'/product/'+params[:id])))['product']
+    
+    #@json_data = JSON.parse('{"date_modified": {"$date": 1451741150716}, "category": "CPU-koelers", "ean": "4260120532850", "Productcode": "TTC-NK85TZ", "_id": {"$oid": "5687c1ce0b5f962489007c20"}, "brand": "Titan", "records": [], "Toegevoegd aan prijsvergelijker": "vrijdag 5 juni 2009",
```
Comment (robinsiep) [inline app/controllers/dashboard_controller.rb]: commented code

File: app/controllers/search_dashboard_controller.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+			json = JSON.parse(Net::HTTP.get(URI.parse('http://95.85.12.99:6543/product?searchterm='+params[:zoekterm]+'&limit=100&offset=0&for_sale=true')))
+			p json.length
+
+			@products = json[0]['products']
+			(1..(json.length-1)).each do |i|
+				@products = @products+json[i]['products']
+			end
+			render json: @products
+
+			#p @products
```
Comment (robinsiep) [inline app/controllers/search_dashboard_controller.rb]: commented code

**Comment by robinsiep** [comment]
Other than my comments there's a failed test and there's conflicts in this branch.

**Pull request closed without merging**